diff --git a/node_modules/ethers/utils/web.js b/node_modules/ethers/utils/web.js
index 05406b4..bd9b19e 100644
--- a/node_modules/ethers/utils/web.js
+++ b/node_modules/ethers/utils/web.js
@@ -81,7 +81,7 @@ function fetchJson(connection, json, processFunc) {
             if (request.status != 200) {
                 cancelTimeout();
                 // @TODO: not any!
-                var error = new Error('invalid response - ' + request.status);
+                var error = new Error(request.responseText || 'invalid response - ' + request.status);
                 error.statusCode = request.status;
                 if (request.responseText) {
                     error.responseText = request.responseText;
